composer.json

```diff
@@ -18,11 +18,11 @@
         "bin/zf-development-mode"
     ],
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^7.0 || ^7.1"
```

Comment:
Sorry,
pushed to wrong repository :-(
I'm quite new to all this stuff.
reverted with https://github.com/zfcampus/zf-development-mode/pull/31/commits/f6526bb1a011df9ce3cf901e8924c972f1ff1ee1